### ISSUE TITLE
Null safety

### ISSUE THREAD

**Issue by jhass** [synw/geojson#37]
Could you please migrate to Dart's new null safety? https://dart.dev/null-safety/migration-guide

**Comment by fryette** [comment]
If you want to use latest version of flutter_map from git, use next workaround
```
  geodesy: ^0.4.0-nullsafety.0
  geopoint:
    git:
      url: https://github.com/thisplaceapp/geopoint.git
      ref: master
      ```

**Comment by synw** [comment]
Could someone help on this? I have no time for it actually

**Comment by it-sam** [comment]
@synw Let me try.

**Comment by bmason8** [comment]
flutter_map just released a null-safe version (0.13). Is this package likely to get updated to null safe soon? Thanks.

**Comment by barbalex** [comment]
I had a look at the dependencies. They could be changed from:
```yaml
dependencies:
  extra_pedantic: ^1.2.0
  geodesy: ^0.3.2
  geopoint: ^0.8.0
  iso: ^0.3.0
  meta: ^1.1.8
  pedantic: ^1.9.2

dev_dependencies:
  test: ^1.6.5
  test_coverage: ^0.4.1
```
to:
```yaml
dependencies:
  extra_pedantic: ^1.5.0
  geodesy: ^0.4.0-nullsafety.0
  geopoint2: ^0.9.0
  iso: ^1.0.0
  meta: ^1.4.0
  pedantic: ^1.11.0

dev_dependencies:
  test: ^1.17.5
  test_coverage: ^0.4.2 # NOT null-safe. See: https://github.com/pulyaevskiy/test-coverage/pull/39
```

The only dependency not null safe is test_coverage. Not sure if that is a problem.

**Comment by it-sam** [comment]
By the time the  [pull request](https://github.com/synw/geopoint/pull/8) get approved, 
There is an alternative.
Add this into your **yaml** file.

**For Geo Json:** 

    dependency_overrides:

	  geojson:
	    git:
  
	      url: https://github.com/it-sam/geojson.git
      
	      commit: 6fa3b9
      
  **For Geo point**
  
  
	  dependency_overrides:
  
	  geopoint:
  
	    git:
    
	      url: https://github.com/it-sam/geopoint.git
      
	      commit: 7a28e2

**Comment by it-sam** [comment]
@synw @jhass We can close now. it has been released in 1.0 version.
